build/gulpfile.mixin.js

```diff
@@ -25,12 +26,63 @@ gulp.task('mixin', function () {
 		return;
 	}
 
-	// {{SQL CARBON EDIT}}
+	// {{SQL CARBON EDIT}} - apply ADS insiders values if needed
 	let serviceUrl = 'https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery.json';
+	let nameShort = product.nameShort;
+	let nameLong = product.nameLong;
+	let applicationName = product.applicationName;
+	let dataFolderName = product.dataFolderName;
+	let win32MutexName = product.win32MutexName;
+	let win32DirName = product.win32DirName;
+	let win32NameVersion = product.win32NameVersion;
+	let win32RegValueName = product.win32RegValueName;
+	let win32AppId = product.win32AppId;
+	let win32x64AppId = product.win32x64AppId;
+	let win32UserAppId = product.win32UserAppId;
+	let win32x64UserAppId = product.win32x64UserAppId;
+	let win32AppUserModelId = product.win32AppUserModelId;
+	let win32ShellNameShort = product.win32ShellNameShort;
+	let darwinBundleIdentifier = product.darwinBundleIdentifier;
+
 	if (quality === 'insider') {
+		let dashSuffix =  '-insiders';
+		let dotSuffix =  '.insiders';
+		let displaySuffix = ' - Insiders';
+
 		serviceUrl = `https://sqlopsextensions.blob.core.windows.net/marketplace/v1/extensionsGallery-${quality}.json`;
+		nameShort += dashSuffix;
+		nameLong += displaySuffix;
+		applicationName += dashSuffix;
+		dataFolderName += dashSuffix;
+		win32MutexName += dashSuffix;
+		win32DirName += displaySuffix;
+		win32NameVersion += displaySuffix;
+		win32RegValueName += dashSuffix;
+		win32AppId = "{{9F0801B2-DEE3-4272-A2C6-FBDF25BAAF0F}";
+		win32x64AppId = "{{6748A5FD-29EB-4BA6-B3C6-E7B981B8D6B0}";
+		win32UserAppId = "{{0F8CD1ED-483C-40EB-8AD2-8ED784651AA1}";
+		win32x64UserAppId += dashSuffix;
+		win32AppUserModelId += dotSuffix;
+		win32ShellNameShort += displaySuffix;
+		darwinBundleIdentifier += dotSuffix;
 	}
+
 	let newValues = {
```

Comment:
nit: This could all be made shorter by just setting the new values and then overwriting them if quality === 'insider'

```
let newValues = {
   "nameShort": nameShort
   ….
}; 

if(quality === 'insiders') {
    newValues.nameShort += dashSuffix;
    ….
};
```